docs/guides/high-availability/single-cluster/building-blocks.adoc

```diff
@@ -22,7 +22,9 @@ include::../partials/blueprint-disclaimer.adoc[]
 [#single-cluster-blocks-low-latency]
 == Multiple availability-zones with low-latency connection
 
-Ensures that synchronous replication is available for both the database and {project_name} clustering.
+A low latency ensures that synchronous replication is available for both the database and {project_name} clustering.
```

Comment:
This sentence was already present before, but I don't think it's clear what we mean by "synchronous replication is available". Maybe we can elaborate with something like:

"A low latency connection ensures that synchronous replication of data, required by the database and {project_name} clustering, does not adversely impact {project_name} response times."

This applies to multi-cluster as well.

Reply:
Next try: 

> {project_name} requires a low latency network connection for the synchronous replication of data by the database and {project_name} clustering.

WDYT?